Keyboard sirius/uni660/rev2/ansi compile error

I tried to compile keyboard sirius/uni660/rev2/ansi on our qmk configurator UI + my windows machine + my mac machine and all received an this error:

```
Linking: .build/sirius_uni660_rev2_ansi_layout_ansi_mine.elf                                        
 | 
 | .build/obj_sirius_uni660_rev2_ansi_layout_ansi_mine/matrix.o: In function `matrix_init':
 | /qmk_compiler/qmk_firmware/keyboards/sirius/uni660/rev2/matrix.c:134: undefined reference to `serial_init'
 | collect2: error: ld returned 1 exit status
 | 
```

I've noticed that the header file containing serial_init() function is included in matrix.c, so I don't know why we have undefined reference here without a deep dive. I've also contacted the author of this keyboard and he is not sure as well and he said it was working last year. Maybe it's related with the way "qmk compile" command works and it has changed since last year?

Look forward to hearing from you, thank you for your help!

--- a/keyboards/sirius/uni660/rev1/rules.mk
+++ b/keyboards/sirius/uni660/rev1/rules.mk
@@ -24,4 +24,4 @@ UNICODE_ENABLE = yes        # Unicode
 CUSTOM_MATRIX = yes
 
 # project specific files
-SRC += matrix.c
+SRC += matrix.c serial_uart.c

--- a/keyboards/sirius/uni660/rev2/rules.mk
+++ b/keyboards/sirius/uni660/rev2/rules.mk
@@ -24,6 +24,6 @@ UNICODE_ENABLE = yes        # Unicode
 CUSTOM_MATRIX = yes
 
 # project specific files
-SRC += matrix.c
+SRC += matrix.c serial_uart.c
 
 DEFAULT_FOLDER = sirius/uni660/rev2/ansi

